### PR TITLE
Release 2017.8

### PR DESCRIPTION
**Pull request by cgwalters**
I think we should do this before we land a lot more stuff; in particular I'd like to release:

 - https://github.com/ostreedev/ostree/pull/989
 - https://github.com/ostreedev/ostree/pull/987

### CODE REVIEW

**Comment by jlebon** [comment]
And maybe https://github.com/ostreedev/ostree/pull/993 as well? It looks like a straightforward useful fix.

File: src/libostree/libostree-devel.sym [hunk 1]
```diff
@@ -18,10 +18,6 @@
 ***/
 
 /* Add new symbols here.  Release commits should copy this section into -released.sym. */
-LIBOSTREE_2017.8 {
-global:
-  ostree_validate_remote_name;
-} LIBOSTREE_2017.7;
```
Comment (jlebon) [inline src/libostree/libostree-devel.sym]: It looks like you may also have to line up the hunk for the next release as well.

Comment (cgwalters) [inline src/libostree/libostree-devel.sym]: Done :arrow_down:

**Comment by jlebon** [comment]
This looks 👍. Just waiting for Homu to be back online and get #993 in.

**Comment by jlebon** [comment]
@rh-atomic-bot r+ 5b99347

**Pull request closed without merging**